defi/src/api2/routes/dimensions.ts

```diff
@@ -263,4 +263,60 @@ export async function getDimensionProtocolFileRoute(req: HyperExpress.Request, r
   if (excludeTotalDataChart) data.totalDataChart = []
   if (excludeTotalDataChartBreakdown) data.totalDataChartBreakdown = []
   return successResponse(res, data)
+}
+
+export async function getDimensionBreakdownProtocolFileRoute(req: HyperExpress.Request, res: HyperExpress.Response) {
+  const protocolName = req.path_parameters.name?.toLowerCase()
```

Comment:
can you add validation that adapterType is fee here, and since it is monthly/quarterly report, I want it to include all datatypes, this way, when the front-end shows financial report of the given protocol it fetches everything in a single call